MAUI/Radio-Button/Visual-Customization.md

```diff
@@ -109,6 +109,20 @@ You can customize the display text appearance of the [`SfRadioButton`](https://h
 
 ![.NET MAUI Radio Button TextAppereance](Images/VisualCustomization/textappearance.png)
 
+## LineBreakMode
+
+The `LineBreakMode` allows you to wrap or truncate the text. The default value of this property is NoWrap. The following other options are available in `LineBreakMode`:
+
+*   `NoWrap` - Avoids the text wrap.
+*   `WordWrap` - Wraps the text by words.
+*   `CharacterWrap` - Wraps the text by character.
+*   `HeadTruncation` - Truncates the text at the start.
+*   `MiddleTruncation` - Truncates the text at the center.
+*   `TailTruncation` - Truncates the text at the end.
+
+This demo can be downloaded from this [link](https://www.syncfusion.com/downloads/support/directtrac/general/ze/RadioButton_11858718544)
```

Comment:
@SwethavasanthSF3977, add xaml and c# code.

Reply:
Added xaml and c# code